mindsdb_native/libs/helpers/conformal_helpers.py

```diff
@@ -4,7 +4,12 @@
 
 import pandas as pd
 import numpy as np
-import copy
+from copy import deepcopy
+
+
+def softmax(x, T=1, axis=0):
```

Comment:
maybe softmax implemented in pytorch can be used instead

Reply:
Yup, I agree. Initially opted for not adding PyTorch as a dependency, but now that we're removing Ludwig there's really no harm in doing so, right?

Reply:
Pytorch install is painful, so don't add it to the requirements.txt

But it's already a dep of lightwood and will always be, so feel free to just import it without having it in the requirements.